SharedCode/build.gradle.kts

```diff
@@ -25,11 +29,20 @@ android {
             isMinifyEnabled = false
         }
     }
+
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
+    }
+
+}
+
+tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile> {
```

Comment:
this was needed in order to inline bytecode for the android app. [kodein github issue](https://github.com/Kodein-Framework/Kodein-DI/issues/257#issuecomment-541847041)